vendor/github.com/containers/image/v5/docker/docker_transport.go

```diff
@@ -43,30 +45,51 @@ func (t dockerTransport) ValidatePolicyConfigurationScope(scope string) error {
 
 // dockerReference is an ImageReference for Docker images.
 type dockerReference struct {
-	ref reference.Named // By construction we know that !reference.IsNameOnly(ref)
+	ref             reference.Named // By construction we know that !reference.IsNameOnly(ref)
```

Comment:
Actually `newReference` now enforces `IsNameOnly` for `isUnknownDigest`, so the comment needs updating.